glide.lock

```diff
@@ -41,7 +41,7 @@ imports:
   - httputil
```

Comment:
Just did `glide up` to include all newly added dependencies.
Didn't we need this?